students/EricAdams/session07/html_render.py

```diff
@@ -35,12 +40,45 @@ def write_to_file(self, file_obj, stuff_to_print):
 
 class Body(Element):
     tag = 'body'
+    indent = '    '
```

Comment:
you don't want to redefine this here. -- unless you want each type of Element to have a different indent level -- probably not. If you define indent only in the base class, then all subclasses will use the same one.

then you can change it in the base class, and the change is seen everywhere.